reference/orders.v2.oas2.yml

```diff
@@ -4468,7 +4464,7 @@ components:
               example: '2001:db8:3333:4444:5555:6666:7777:8888'
               maxLength: 39
             is_deleted:
-              description: Indicates whether the order was deleted (archived). Set to to true, to archive an order.
+              description: This read-only parameter indicates if the order was deleted or archived.
```

Comment:
@bc-tgomez Can you check but I think you might need to delete this one?  In this case, the `is_deleted` field is inside order_Put which is used in a request body.